### ISSUE TITLE
Fetch new notifications 

### ISSUE THREAD

**Issue by PavanTeja2005** [cro2003/rgpvApi#19]
I'm thinking of adding a feature to fetch latest notifications from the college website.

**Comment by cro2003** [comment]
Sure, in notiofication what will be the attributes, like name, description etc.

**Comment by PavanTeja2005** [comment]
I'm think of fetching these.

![image](https://github.com/user-attachments/assets/73c0b0a3-73b8-4437-a306-6af1b7cbd093)


and provide downloadable links. Or extracting data from pdfs.

**Comment by cro2003** [comment]
Yes, but it would be great if you can get notification from here: https://www.rgpv.ac.in/Uni/ImpNoticeArchive.aspx, there you will get Notification title, date and link to the pdf. Then the notification function will take two optional argument from date & to date and it will return the notification released between these days, if no arguments are passed then the notification from previous 30 days will be showed.

**Comment by PavanTeja2005** [comment]
Okay, thank you for the suggestion. I will work on it

**Comment by PavanTeja2005** [comment]
Please assign me with it and provide the labels
